packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -532,16 +532,28 @@ module.exports.external = [
   },
   {
     name: "multipleToArray",
-    description: "Returns an array combining all given objects: multipleToArray obj1 obj2 …",
     func: function (...vals) {
-        var combinedArr = [];
-        vals.forEach(function(val) {
-            if (Array.isArray(val)) {
-                combinedArr.push(...val);
-            } else if (val) {
+        const uniqueSet = new Set();
+        const combinedArr = [];
+
+        // Use vals.length - 1 to exclude the last argument
+        for (let i = 0; i < vals.length - 1; i++) {
```

Comment:
excluding last argument which is the whole message and makes the array one longer than its supposed to be